partcad-cli/src/partcad_cli/click/commands/list/mates.py

```
@@ -0,0 +1,126 @@
+            if recursive and not package_name.startswith(package):
+                continue
+
+            package = ctx.projects[package_name]
+            for interface_name in package.interfaces:
+                package.get_interface(interface_name)
+
+        output = "PartCAD mating interfaces:\n"
+        for source_interface_name in ctx.mates:
+            source_package_name = source_interface_name.split(":")[0]
+            # TODO-102: @alexanderilyin: Use interface short name
+            short_source_interface_name = source_interface_name.split(":")[1]
```

Ruff (0.8.2)

Local variable `short_source_interface_name` is assigned to but never used

Remove assignment to unused variable `short_source_interface_name`

(F841)